experiments/grafana/docker-compose.yml

```diff
@@ -39,6 +48,15 @@ services:
       # Transport protocol
       - "4300:4300"
     command: -Ccluster.name=democluster -Chttp.cors.enabled=true -Chttp.cors.allow-origin="*"
+    deploy:
```

Comment:
if this is to be used with docker compose, deploy configuration does not make sense.
if this is to be used with docker stack, this configuration, using global will create an instance on each node, but the instances won't be in synch

Reply:
Right, you've got a point. It's currently being used with compose, so the deploy part is ignored. Adding a volume made me automatically add the "one per node" constraint, but sure, this won't be either enough to make it work with stack deploy. For that I'll push the other recipe. I just didn't want to leave this one as it was without persistence.

Responding your questions, removing a stack would typically not remove the volume. Deploying a stack when the volume was previously created will use the previously created volume.

Reply:
then i recommend either to make the compose script using a deploy with replica 1 (so that i can be used for testing using `docker stack deploy`) or to remove the deploy part

Reply:
Ok, removed deploy part to avoid confusion